creativecommons/mp
# Resolving the text overflow in Divs ( "What comes after implementation" and "About" section)

## Fixes
fixes #76 by @shubhanshu02
## Description
The PR resolves the text overflow occuring in the Div section i.e
 - What comes after implementation
 - About
 
When viewed on devices with a width between 768px to around 1300px.
## Technical Details
changed some hardcoded widths and heights from these cards , for the pt-card and pt-text-card classes:



## Tests
1) Open the "ABOUT" section of the page.
2) Adjust the width size between 768px to 1300px.
3) Notice that the text overflow which was occuring previously is rectified.
4) Follow step 1 and 2 for "WHAT COMES AFTER IMPLEMENTATION" section.

## Screenshot
![Screenshot (167)](https://user-images.githubusercontent.com/72491412/107843804-0fd50a80-6df4-11eb-819e-dec8f4e5064e.png)

![Screenshot (169)](https://user-images.githubusercontent.com/72491412/107843863-8ffb7000-6df4-11eb-8b63-cc6d9d59043b.png)





## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
